From e384deff3d3a56c97bd11c8f0ef432fd1632914e Mon Sep 17 00:00:00 2001
From: Sojiro Kusunoki <sojiro.kusunoki.zj@ps.hitachi-solutions.com>
Date: Thu, 13 Sep 2018 19:26:36 +0900
Subject: [PATCH 638/909] arm64: dts: r8a77995-draak: Add definition for
 vspm/vsp2drifer

commit 69e56c10580bcd97c6e03d69aaf1797bc8b335fe from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Signed-off-by: Sojiro Kusunoki <sojiro.kusunoki.zj@ps.hitachi-solutions.com>
Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77995-draak.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
index e6a743079e88..2e01b79c1651 100644
--- a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
@@ -493,3 +493,7 @@
 		};
 	};
 };
+
+&vspbs {
+	status = "okay";
+};
-- 
2.17.1

